src/mlpack/methods/kde/kde_model.hpp

```diff
@@ -227,6 +396,22 @@ class KDEModel
   //! Type of tree.
   TreeTypes treeType;
 
+  //! Whether Monte Carlo estimations will be used.
+  bool monteCarlo;
+
+  //! Probability of estimation being bounded by relative error when using
+  //! Monte Carlo estimations.
+  double mcProb;
+
+  //! Size of the initial sample for Monte Carlo estimations.
+  size_t initialSampleSize;
+
+  //! Entry coefficient for Monte Carlo estimations.
+  double mcEntryCoef;
+
+  //! Break coefficient for Monte Carlo estimations.
+  double mcBreakCoef;
```

Comment:
I see, having these held in `KDEModel` instead of `KDE` can make access a little bit easier, but if we're going to do that we should add a comment pointing out that these are held to simplify the code of accessing or changing these values for the underlying `KDE<>` object.

There is some risk of these things being out of sync, but you are using it carefully in the rest of the code, so I think it is not a problem.